docs/examples/frameworks/closed_single_supplier.json

```diff
@@ -0,0 +1,50 @@
+{
+  "uri": "https://example.com/ocds/ocds-r6ebe6-9af22e2f-5307-4e98-bff6-1ebffbff343d/2020_S110-268595",
+  "version": "1.1",
+  "publishedDate": "2020-06-05T00:00:00Z",
+  "publisher": {
+    "name": "Open Contracting Partnership"
+  },
+  "license": "https://creativecommons.org/publicdomain/zero/1.0/",
+  "publicationPolicy": "http://example.com/publicationPolicy",
+  "releases": [
+    {
+      "ocid": "ocds-r6ebe6-9af22e2f-5307-4e98-bff6-1ebffbff343d",
+      "id": "2020/S 110-268595",
+      "date": "2020-06-05T00:00:00Z",
+      "tag": [
+        "award"
+      ],
+      "initiationType": "tender",
+      "tender": {
+        "id": "NP45719",
+        "status": "complete"
+      },
+      "parties": [
+        {
+          "name": "Movianto UK",
+          "id": "1",
+          "roles": [
+            "supplier"
+          ]
+        }
+      ],
+      "awards": [
+        {
+          "id": "NP45719",
+          "status": "active",
+          "suppliers": [
+            {
+              "name": "Movianto UK",
+              "id": "1"
+            }
+          ],
+          "value": {
```

Comment:
I noticed that we added a value field here but didn't explain what this value should be

Reply:
Good spot. I don't think we ever discussed whether to set `awards/value` in the first stage. Perhaps omitting it would be best, to reduce the risk of double-counting?

Reply:
Yes, I thought the same, and that was the advice we had given to Chile as well (but also because they didn't have quality data about this). The only use case that we may miss is comparing the price with one the supplier entered into the framework and their subsequent actual awards.  But I agree with we omitting this field at the first stage

Reply:
Removed in https://github.com/open-contracting/standard/pull/1123/commits/3b22cafbd65275b0ae74c01547a0c479e8be2634